sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputationResults.java

```diff
@@ -41,23 +51,34 @@
     @JsonProperty("values")
     private final List<SensitivityValue> sensitivityValues;
 
+    @JsonProperty("contingencies present")
+    private final boolean contingenciesPresent;
+
+    @JsonProperty("contingencies values")
```

Comment:
I know that JSON format allows whitespaces in properties but shouldn't we use camel case instead to be consistent with other results serialization?